Resources/Prototypes/Datasets/tips.yml

```diff
@@ -32,30 +28,99 @@
+  - "As a Cargo Technician, remember that you can order guns in an emergency! The extra firepower can often be the difference between you and your fellow crewmembers living or dying."
   - "As the Bartender, you can use a circular saw on your shotgun to make it easier to store."
+  - "As a Botanist, you can mutate and crossbreed plants together to create more potent produce that also has higher yields."
   - "As the Clown, spice your gimmicks up! Nobody likes a one-trick pony."
   - "As the Clown, if you lose your banana peel, you can still slip people with your PDA! Honk!"
   - "As the Mime, your oath of silence is your source of power. Breaking it robs you of your powers and of your honor."
   - "As the Lawyer, try to negotiate with the Warden if sentences seem too high for the crime."
   - "As a Security Officer, communicate and coordinate with your fellow officers using the security radio channel to avoid confusion."
+  - "As a Security Officer, remember that correlation does not equal causation. Someone may have just been at the wrong place at the wrong time!"
+  - "As a Detective, you can chase criminals more effectively by using fingerprint and fiber data obtained from forensic scans of objects the perpetrator likely interacted with."
   - "As an Atmospheric Technician, your ATMOS holofan projector blocks gases while allowing objects to pass through. With it, you can quickly contain gas spills, fires and hull breaches."
+  - "As an Atmospheric Technician, try to resist the temptation of making canister bombs for Nuclear Operatives, unless you're in a last-ditch scenario. They often lead to large amounts of unnessecarry friendly fire!"
   - "As an Engineer, you can repair windows by using a welding tool on them while not in combat mode."
   - "As an Engineer, you can electrify grilles by placing powered cables beneath them."
+  - "As an Engineer, you can use plasma glass to reinforce an area and prevent radiation. Uranium glass can also be used to prevent radiation."
   - "As the Captain, you are one of the highest priority targets on the station. Everything from revolutions, to nuclear operatives, to traitors that need to rob you of your unique laser gun or your life are things to worry about."
   - "As the Captain, always take the nuclear disk and pinpointer with you every shift. It's a good idea to give one of these to another head you can trust with keeping it safe."
   - "As the Captain, you have absolute access and control over the station, but this does not mean that being a horrible person won't result in mutiny."
+  - "As the Captain, try to be active and patrol the station. Staying in the bridge might be tempting, but you'll just end up putting a bigger target on your back!"
+  - "As a Scientist, you can try random things on an artifact while the scanner is on cooldown to speed up the point extraction process significantly."
```

Comment:
This doesn't help in the long-term, since you then can't use the node scanner and prints to skip scanning.

Reply:
i've heard emogarbage suggest it was a good method to speedrun science, but i'll take your word for it

Reply:
It does work, and Emo did make artifacts.